lib/adapter/TingClientRequestAdapter.php

```diff
@@ -88,19 +95,28 @@ public function execute(TingClientRequest $request) {
       // Logging changed to make more granular logging
       $log = array(
         'action' => $soapAction,
-        'requestBody' => $this->client->requestBodyString,
+        'adapter' => $this->client->getRequestAdapter(),
+        'requestBody' => $this->client->getRequestBodyString(),
+        'requestVariables' => $this->client->getRequestVariables(),
         'wsdlUrl' => $request->getWsdlUrl(),
       );
       $this->logger->log('soap_request_complete', $log);
-      //$this->logger->log('Completed SOAP request ' . $soapAction . ' ' . $request->getWsdlUrl() . ' (' . round($time, 3) . 's). Request body: ' . $client->requestBodyString);
 
       // check if http_code is a valid url
-      if (method_exists($this->client, 'getCurlInfo')){
+      $curl_info = array();
+      if (method_exists($this->client, 'getCurlInfo')) {
         $curl_info = $this->client->getCurlInfo();
       }
 
-      if($curl_info['http_code'] != 200){
-        throw new TingClientHttpStatusException('Curl returns wrong http code ('.$curl_info['http_code'].')',  $curl_info['http_code'] );
+      if ($curl_info['http_code'] != 200) {
+        $response = json_decode($response);
+        // Open Platform has more detailed error description.
+        $error = $error_description = NULL;
+        if (!empty($response->errorMessage) && $errorMessage = json_decode($response->errorMessage)) {
+          $error = (!empty($errorMessage->error)) ? '. '. $errorMessage->error : NULL;
```

Comment:
Forslag: 
Måske skulle du skrive "tom fejlbesked" i stedet for NULL. Der bliver kastet en exception uden data lige nu?